src/main/java/picard/vcf/MergeVcfs.java

```diff
@@ -54,14 +54,74 @@
 import java.util.List;
 
 /**
- * Combines multiple VCF files into a single file. Input files must be sorted by their contigs
- * and, within contigs, by start position. Throws IllegalArgumentException if the contig lists
- * are not present in the input files, are not identical or if the sample lists are not the
- * same; this class uses the GATK to merge headers, which may throw exceptions if the headers
- * cannot be merged. See VCFUtils.smartMergeHeaders for details.
- * <p/>
- * An index file is created for the output file by default. Using an output file name with a
- * ".gz" extension will create gzip-compressed output.
+ * Combines multiple variant files into a single variant file.
+ *
+ * <h3>Inputs</h3>
+ *  <ul>
+ *      <li>one or more input VCF files in text uncompressed (name ending with ".vcf"), compressed text compressed format (name ending with ".vcf.gz")
+ *          or binary format BCF (name ending with ".bcf"),</li>
+ *      <li>and optionally a sequence dictionary file (typically name ending in .dict) if the input VCF does not contain a
+ *          complete contig list and if the output index is to be created (true by default).</li>
+ *  </ul>
+ *  <p>
+ *  The input variant files must:
+ *     <ul>
+ *         <li>if there are samples, those must be the same across all input files,</li>
+ *         <li>input file headers must be contain compatible declarations for common annotations (INFO, FORMAT fields) and filters.</li>
```

Comment:
Needs comma at end not period.

Reply:
I disagree -- better to make them independent capitalized sentences